core/python_utils.py

```diff
@@ -308,27 +287,6 @@ def url_encode(query, doseq=False):
     return urlparse.urlencode(query, doseq=doseq)
 
 
-def url_open(source_url):
-    """Open a network object denoted by a URL for reading using
-    urllib2.urlopen if run under Python 2 and urllib.request.urlopen if
-    run under Python 3.
-
-    Args:
-        source_url: str. The URL.
-
-    Returns:
-        urlopen. The 'urlopen' object.
-    """
-    # TODO(#12912): Remove pylint disable after the arg-name-for-non-keyword-arg
-    # check is refactored.
-    context = ssl.create_default_context(cafile=certifi.where())  # pylint: disable=arg-name-for-non-keyword-arg
-    try:
-        import urllib.request as urlrequest
-    except ImportError:
-        import urllib2 as urlrequest
-    return urlrequest.urlopen(source_url, context=context)
```

Comment:
I think this should be refactored and moved to _utils.py_. Then use the version from _utils.py_ in the codebase. This is needed because we need the SSL context.

Reply:
Done